ignite/services/plugin/clientAPI.go

```diff
@@ -0,0 +1,40 @@
+package plugin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/ignite/cli/ignite/pkg/cosmosanalysis/chain"
+	chainservice "github.com/ignite/cli/ignite/services/chain"
+)
+
+// NewClientAPI creates a new app ClientAPI.
+func NewClientAPI(c *chainservice.Chain) clientAPI {
```

Comment:
Here you can return the interface instead of a private type:
```suggestion
func NewClientAPI(c *chainservice.Chain) ClientAPI {
```